scripts/complete.sh

```diff
@@ -2,7 +2,7 @@ _learn_ocaml_add_f()
 {
   local cmd
   cmd=$1; shift
-  _learn_ocaml_add "$($cmd "$@" 2>/dev/null)"
+  IFS=$'\n' _learn_ocaml_reply+=("$($cmd "$@" 2>/dev/null)")
```

Comment:
Redefinition of IFS is very fragile. (I am working on the semantics of POSIX shell in the COLIS ANR project and I would not recommend playing with IFS.)